### PR TITLE
template handling set up

### PR DESCRIPTION
**Pull request by jusk02**
Initial set up for templates/components to be loaded in the correct page depending on the data provided

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1,3 +1,52 @@
 import '../scss/main.scss';
+// import '../about.html';
```
Comment (jsebax) [inline src/js/index.js]: It's not necessary to import de HTML file here. If you create the link to the page from the `index.html`, `parcel` compile the pages automatically.

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,9 @@
 </head>
 <body>
   <div id="root"></div>
+
+  <a href="./about.html"></a>
```
Comment (jsebax) [inline src/index.html]: Add some text to this link to view it on the page

**Comment by jsebax** [comment]
For some reason, in the `index.html` the styles are not being loaded, only in the `about.html`. So, to solve this add this `<link rel="stylesheet" href="./scss/main.scss">` in the `index.html` file.

File: src/index.html (hunk 2 above)
Comment (jusk02) [reply src/index.html]: About this, we don't really want this link to be seen, it is just here for parcel to pull in the file on serve/build time

Comment (jsebax) [reply src/index.html]: Got it

**Review by jsebax — APPROVED** [review]

**Review by Frankruiz90 — APPROVED** [review]

**Review by Frankruiz90 — APPROVED** [review]

**Pull request merged**